javascript/cw_tool.referrer.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file
+ *  Add referrer related behaviours.
+ */
+var CW = CW || {};
+
+(function($) {
```

Comment:
The jQuery code styling shows the use of $
https://contribute.jquery.org/style-guide/js/

Do you prefer calling jQuery each time instead of $? I don't care to be honest. If we agree to do that I can change everything from now on.